### PR TITLE
[PDN-656] Update - File upload

### PR DESCRIPTION
**Pull request by RipripKipkip**
image
image
image
image

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1059

File: packages/scss/src/components/inputs/file/_file.scss [hunk 1]
```diff
@@ -1,30 +1,35 @@
 .file {
 	border: 2px dashed _theme("commons.divider.color");
-	border-radius: 3px;
+	border-radius: _theme("commons.border.radius");
 	display: block;
 	max-width: 30rem;
-	padding: .5rem _theme("spacings.standard") 1.5rem;
+	padding: 4rem _theme("spacings.standard") 4rem;
```
Comment (vvalentin-lucca) [inline packages/scss/src/components/inputs/file/_file.scss:6]: 4rem = biggest

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1059

File: packages/scss/src/components/inputs/file/_file.scss (hunk 1 above)
Comment (RipripKipkip) [reply packages/scss/src/components/inputs/file/_file.scss]: thx 👍

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1059

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1059

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1059

File: packages/scss/src/components/inputs/file/_file.scss [hunk 2]
```diff
@@ -37,31 +38,36 @@
 
 .file {
 	&.mod-small {
-		padding: _theme("spacings.smaller") _theme("spacings.standard")
-		_theme("spacings.smaller") 5rem;
+		align-items: center;
+		display: flex;
+		justify-content: space-between;
```
Comment (BertrandPodevin) [inline packages/scss/src/components/inputs/file/_file.scss:43]: Perhaps a `flex-wrap: wrap;` could be added here for a better responsive behavior ?

File: packages/scss/src/components/inputs/file/_file.scss [hunk 3]
```diff
@@ -1,30 +1,31 @@
...
-	margin: 0 0 .5rem;
-	font-size: _theme("sizes.big.font-size");
-}
-
 .file-progress {
-	margin: 1rem 0 .5rem;
+	margin: _theme("spacings.small") 0 _theme("spacings.smaller");
 }
 
 .file-input {
```
Comment (vvalentin-lucca) [inline packages/scss/src/components/inputs/file/_file.scss:31]: Really bad for accessibility and prevents drag and drop too.
We should consider something like this I think: https://codepen.io/vincent-valentin/pen/KKVVomb

Comment (sanlucca) [reply packages/scss/src/components/inputs/file/_file.scss:31]: What is your suggestion here ?

Comment (vvalentin-lucca) [reply packages/scss/src/components/inputs/file/_file.scss:31]: Don't apply a `display: none` to the input but make it almost transparent and positioned over the area. (With a empty title for prevent any native tooltip.)

Comment (sanlucca) [reply packages/scss/src/components/inputs/file/_file.scss:31]: Oh you're commenting something out of the PR that's why I couldn't understand. I agree with you, we should make these adjustments

Comment (vvalentin-lucca) [reply packages/scss/src/components/inputs/file/_file.scss:31]: Yeah, I thought it was an opportunity. But don’t worry, we can do it in another pull request.

**Pull request merged**